packages/next-typescript/index.js

```diff
@@ -1,4 +1,7 @@
-module.exports = (nextConfig = {}) => {
+module.exports = (nextConfig = {}, customs) => {
+
+  const { isTranspileOnly } = customs;
```

Comment:
Lets read from `nextConfig` instead of adding another parameter.

Also call it `typescriptLoaderOptions`

Reply:
@timneutkens before I make the changes I would like to ask something.
I've tried to change it directly from node_modules/@zeit/next-typescript/index.js setting transpileOnly to "false".

But it still doesn't raise any errors, as long as I changed the lib "hard coding it" it should work right?

Can you give me a light please?